apcd-cms/src/apps/admin_regis_table/templates/create_submitter_modal.html

```diff
@@ -39,11 +39,11 @@ <h4>File Submission</h4>
           </article>
         </div>
 
-        <form method="POST" action="" class="c-form">
+        <form method="POST" action="" class="c-form" name="create-submitter-form">
           <h4>Submitter Data</h4>
 
           {% csrf_token %}
-          <input type="hidden" name="_token" value>
+          <input type="hidden" name="create-submitter-form" value>
```

Comment:
Is this intentional?

Reply:
Yep! I am using this to differentiate between different POST forms on this page.